packages/rolldown/package.json

```diff
@@ -0,0 +1,76 @@
+{
+  "name": "rolldown",
+  "version": "0.0.1",
```

Comment:
The package previously has versions published up to 0.3.0 and 0.4.x-alpha, we should start at 0.5.0 just to be safe.